include/mega/megaclient.h

```diff
@@ -1438,6 +1438,14 @@ class MEGA_API MegaClient
     // the SDK is trying to log out
     int loggingout;
 
+    // Account type (pro level)
+    int proLevel;
```

Comment:
Get rid of `proLevel`, since it should be retrieved in the old way, through `uq` command (`getuseraccountdetails()`?)